src/inject/GPMInject/index.js

```diff
@@ -35,7 +35,7 @@ const waitForExternal = setInterval(() => {
       try {
         fn();
       } catch (e) {
-        console.error(e); // eslint-disable-line
+        Logger.error('Emitter fn() threw exception.', e);
```

Comment:
Anything inside "inject" won't have the global logger availible, electron works in two processes (renderer and main), look into this to figure out how you would make them global in both.

There is also a trick in electron that would go something like `require('electron').remote.getGlobal('globalNameHere')`